Sources/QueuesRedisDriver/JobsRedisDriver.swift

```diff
@@ -55,6 +57,26 @@ public struct RedisQueuesDriver {
     ///   - eventLoopGroup: The `EventLoopGroup` to run the driver with
     public init(configuration config: RedisConfiguration, on eventLoopGroup: EventLoopGroup) {
         let logger = Logger(label: "codes.vapor.redis")
+        let eventLoop = eventLoopGroup.next()
+        let redisTLSClient: ClientBootstrap? = {
+            guard let tlsConfig = config.tlsConfiguration,
+                    let tlsHost = config.tlsHostname else { return nil }
+
+            return ClientBootstrap(group: eventLoop)
+                .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
+                .channelInitializer { channel in
+                    do {
+                        let sslContext = try NIOSSLContext(configuration: tlsConfig)
+                        return EventLoopFuture.andAllSucceed([
+                            channel.pipeline.addHandler(try NIOSSLClientHandler(context: sslContext,
+                                                                                serverHostname: tlsHost)),
+                            channel.pipeline.addBaseRedisHandlers()
+                        ], on: channel.eventLoop)
+                    } catch {
+                        return channel.eventLoop.makeFailedFuture(error)
+                    }
```

Comment:
A couple items:
- We can factor out the creation of the `NIOSSLClientHandler` into a separate closure so we don't have to wrap the whole mess in a `do`/`catch` block; this also lets us use `EventLoop.makeCompletedFuture(withResultOf:)` to avoid the explicit `do`/`catch` altogether.
- Using `EventLoopFuture.andAllSucceed(_:on:)` does not guarantee the order the futures will actually run in; instead, since we're on the channel's event loop, just use the pipeline's sync operations to add the SSL handler and chain that to adding the base Redis handlers.
```suggestion
                    channel.eventLoop.makeCompletedFuture {
                        try channel.pipeline.syncOperations.addHandler(NIOSSLClientHandler(
                            context: NIOSSLContext(configuration: tlsConfig),
                            serverHostname: config.tlsHostname
                        ))
                    }
                    .flatMap { channel.pipeline.addBaseRedisHandlers() }
```